### PR TITLE
Expose application logs.

### PR DESCRIPTION
**Pull request by smn**
Each follower is running https://github.com/universalcore/logdriver on port 3000.
The logs are available on the following URL path:

stdout:

```
http://<host>:3000/tail/<slave-id>/frameworks/<framework-id>/executors/<task-id>/runs/latest/stdout
```

stderr:

```
http://<host>:3000/tail/<slave-id>/frameworks/<framework-id>/executors/<task-id>/runs/latest/stderr
```
- [x] introspect the application
- [x] find the task-ids
- [x] find out the slave-ids
- [x] find the framework-ids
- [x] Have Django view that generates a response with an [`X-Accel-Redirect`](http://wiki.nginx.org/X-accel) header set pointing at the correct URL.
- [x] Write a client facing page that uses [HTML5s EventSource](http://www.html5rocks.com/en/tutorials/eventsource/basics/) & javascript to connect to the Django view for the app and in realtime prints the logs in the page somewhere.
- [x] Have Nginx proxy the output from logdriver straight to Javascript.

Notes: 
- There can be multiple tasks for an app, all the logs will need to be event-sourced.
- Logdriver doesn't provide a way to combine these, nor does it append metadata about the server the logs originate from, we may want to add that?

### CODE REVIEW

**Comment by smn** [comment]
Just added some templates, js & styling stubby stuff. Not ready for review and happy for anyone to take & run with it.

File: unicoremc/static/js/app_logs.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+    var al = new AppLog('#logs', function (msg) {
+        $("<div class='entry'/>").text(msg).appendTo(this.target);
+    });
+
+    window.setInterval(function () {
+        al.log(new Date());
+    }, 500);
```
Comment (smn) [inline unicoremc/static/js/app_logs.js]: This is to mock out the EventSource-y stuff.

**Comment by smn** [comment]
logs

Work in progress, the grey lines are stdout, the red lines are stderr. Using fake data atm.

**Comment by smn** [comment]
@miltontony & @Rizziepit ready for review.

**Comment by miltontony** [comment]
:+1: awesome

**Pull request merged**